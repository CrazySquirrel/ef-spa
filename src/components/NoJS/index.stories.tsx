import * as React from 'react';

import {storiesOf} from '@storybook/react';

import NoJs from './index';

storiesOf('NoJs', module)
    .add('default', () => (
        <NoJs/>
    ));
