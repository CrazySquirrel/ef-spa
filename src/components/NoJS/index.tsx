import * as React from 'react';
import * as bem from 'bem-cn';

export default class NoJS extends React.Component {
  public render() {
    const block = bem('no-js');

    return (
        <noscript className={block()}>
          You must enable JavaScript for the full operation of this site.
        </noscript>
    );
  }
}
