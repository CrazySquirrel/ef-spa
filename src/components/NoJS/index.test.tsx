import * as React from 'react';
import {shallow} from 'enzyme';

import NoJS from './index';

describe('NoJS', () => {
  it('render', () => {
    const result = shallow(<NoJS/>);

    expect(result).toMatchSnapshot();
  });
});
