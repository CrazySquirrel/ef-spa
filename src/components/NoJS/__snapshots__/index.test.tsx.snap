version https://git-lfs.github.com/spec/v1
oid sha256:a341d7795fc3fd3399791600d908bd5e791a4d754693ca5fc254ff9116468d44
size 3317
